templates/base.html

```diff
@@ -87,29 +87,43 @@
 
     <a href="{{ url('home') }}" class="logo">{{ _('Mozilla Developer Network') }}</a>
 
-    <div class="header-login">
+    <div id="nav-sec">
         {% if not is_sphinx %}
-            {% include "includes/login.html" %}
+
+            {% if waffle.flag('header_tools') %}
+                  <ul>
+                  {% if user.is_authenticated() %}
+                      <li class="nav-tools"><a href="{{ wiki_url('Mozilla/Connect') }}">{{ _('Tools') }}<i aria-hidden="true" class="icon-caret-down"></i></a>
+
+                      <div class="submenu submenu-cols-2 js-submenu" id="nav-tools-submenu">
+                        <div class="submenu-column">
+                          <div class="title">{{ _('Contribute to MDN') }}</div>
+                          <ul>
+                            <li><a href="{{ wiki_url('Web/HTML') }}">{{ _('Get started') }}</a></li>
+                            <li><a href="{{ wiki_url('Web/CSS') }}">{{ _('Find a task to do') }}</a></li>
+                            <li><a href="{{ wiki_url('Web/JavaScript') }}">{{ _('Review articles') }}</a></li>
+                            <li><a href="{{ wiki_url('Web/Guide/Graphics') }}">{{ _('Add a new page') }}</a></li>
+                          </ul>
+                        </div><div class="submenu-column last">
+                          <div class="title">{{ _('Site Tools') }}</div>
+                          <ul>
+                            <li><a href="{{ wiki_url('Learn') }}">{{ _('Revision dashboard') }}</a></li>
+                            <li><a href="{{ wiki_url('Web/Tutorials') }}">{{ _('Documentation status') }}</a></li>
+                            <li><a href="{{ wiki_url('Web/Reference') }}">{{ _('Localization status') }}</a></li>
+                          </ul>
+                        </div>
+                      </div>
+                   </li>{% endif %}<li>{% include "includes/login.html" %}</li>
```

Comment:
Sadly, can't all be on the same line. Because white space is hard :(

This is the technique we're using to combat the white space problems currently, another option would be to comment out the white space, I can make the switch if we think it would be more readable.

```
<li>stuff</li><li>stuff</li>
```
vs
```
<li>stuff</li><!--
--><li>stuff</li>
```